.gitignore

```diff
@@ -21,5 +24,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-# vim 
```

Comment:
oops :/

Reply:
haha not sure why that got commented out :)

Reply:
oh it's just a header explaining the following lines... so i don't accidentally commit my swap files that are generated by vim

Reply:
i did not mean to add the trailing white-space though